pl_bolts/metrics/object_detection.py

```diff
@@ -22,7 +22,8 @@ def giou(preds: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
         tensor([[-0.0794]])
 
     Returns:
-        GIoU value
+        GIoU tensor: an NxM tensor containing the pairwise GIoU values for every element in preds and target,
+                     where N is the number of prediction bounding boxes and M is the number of target bounding boxes
```

Comment:
```suggestion
        GIoU tensor: an NxM tensor containing the pairwise GIoU values for every element in preds and target,
            where N is the number of prediction bounding boxes and M is the number of target bounding boxes
```
otherwise, it will look like:
![20201224-135425](https://user-images.githubusercontent.com/20610905/103061716-8cffb280-45ef-11eb-9464-478a9e475794.png)

Reply:
Sorry, this suggestion wasn't correct.